tests/darwin/dataset/remote_dataset_test.py

```diff
@@ -398,7 +398,7 @@ def test_works_on_videos(
                             "height": 1080,
                             "thumbnail_url": "",
                             "source_files": [
-                                {"file_name": "test_video.mp4", "url": ""}
+                                {"file_name": "test_video/0000000.png", "url": ""}
```

Comment:
This is necessary due to the change in this PR to always read images from `source_files.file name` instead of `item.name`. In this test on the master branch, you'll see that `item.name` [already points to the PNGs](https://github.com/v7labs/darwin-py/blob/384f1efd3f01862ecf621ebe110d6f8e30f20cd0/tests/darwin/dataset/remote_dataset_test.py#L379) (split video frames), so given the change we also need to point source_files.file_name to the PNGs